src/config/schema.rs

```diff
@@ -117,6 +120,25 @@ pub struct S3 {
     pub secret_access_key: String,
     pub endpoint: Option<String>,
     pub bucket: String,
+    pub cache_properties: Option<ObjectCacheProperties>,
+}
+
+#[derive(Deserialize, Debug, PartialEq, Eq, Clone)]
+#[serde(default)]
+pub struct ObjectCacheProperties {
+    pub capacity: u64,
+    pub min_fetch_size: u64,
+    pub ttl: u64, // We could use humantime_serde crate to parse directly into `std::time::Duration`
```

Comment:
minor: could call it `ttl_s` to make it explicit that we expect a TTL in seconds.